api/app/use_cases/analysis/tree_cover_loss/tree_cover_loss_service.py

```diff
@@ -0,0 +1,13 @@
+from app.models.land_change.tree_cover_loss import TreeCoverLossAnalyticsIn
+from app.models.common.analysis import AnalysisStatus
+
+
+class TreeCoverLossService:
+    def __init__(self, utils=None):
+        pass
+
+    def do(self, tree_cover_loss_analytics: TreeCoverLossAnalyticsIn):
+        pass
```

Comment:
Were you wanting to just merge in the empty interface and follow up with the actual computation?